Code/GraphMol/MolHash/mf.h

```diff
@@ -0,0 +1,275 @@
+/*==============================================*/
+/* Copyright (C)  2016-2019  NextMove Software  */
+/* All rights reserved.                         */
+/*                                              */
+/* This file is part of molhash.                */
+/*                                              */
+/* The contents are covered by the terms of the */
+/* BSD license, which is included in the file   */
+/* license.txt.                                 */
+/*==============================================*/
+#ifndef NMS_MOLFORMULA_H
+#define NMS_MOLFORMULA_H
+
+static unsigned char OrganicHillOrder[119] = {
```

Comment:
Use const not static, static forces the compiler to create a unique copy for every source file that reads this header.

Reply:
yes. Otherwise you end up with multiple definitions of these variables if you include mf.h in multiple files. Which nobody should be doing anyway.

Reply:
I see that it is include in hashfunctions.cpp and nothing else.  Does it need to be in the rdkit_headers in CMakeLists?  Either way, I happy to let this one go.

Reply:
That's a good point. It's not actually needed in client code. I will fix that.